### PR TITLE
feat: add toast, grid and list

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds three more components into component library -
- Toast
- Simple grid
- Lists

https://deploy-preview-12--component-vishalg8454.netlify.app/

image

image

image

### CODE REVIEW

File: components/toast.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    max-width: 25rem;
+    min-width: 20rem;
+}
+.toast-success{
+    color: var(--color-dark-green);
+    background-color: var(--color-light-green);
+}
+.toast-warning{
+    background-color: var(--color-light-yellow);
+    color: var(--color-dark-yellow);
```
Comment (arjun-computer-geek) [inline components/toast.css:14]: Great you have used colour variables.

**Review by arjun-computer-geek — APPROVED** [review]
Good job @vishalg8454 . Code is cleaner and readable.

**Review by Developers-codz — APPROVED** [review]
Good work both on UI and code, approving it for merge

File: components/toast.css (hunk 1 above)
Comment (vishalg8454) [reply components/toast.css:14]: Thanks.

**Review by Shailseen — COMMENTED** [review]
You can add more styles of grid like 50 : 50, 70 :30 ,30 : 70.
code looking clean to me.

**Pull request merged**